packages/cli/src/lib/cmds/doctor.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {ConsoleLog, Log} from '@bubblewrap/core';
+import {join} from 'path';
+import {existsSync, promises as fsPromises} from 'fs';
+import {loadOrCreateConfig} from '../config';
+import {enUS as messages} from '../strings';
+
+async function jdkDoctor(log: Log): Promise<boolean> {
+  const config = await loadOrCreateConfig();
+  const jdkPath = config.jdkPath;
+  const file = await fsPromises.readFile(join(jdkPath, 'release'), 'utf-8');
```

Comment:
nit: the variable name is a bit deceiving - this not a file, but the content of the `release` file.